peekaboo/toolbox/cortex.py

```diff
@@ -518,15 +524,17 @@ def set_ok(self, cookie, request):
 
 class Cortex:
     """ Interfaces with a Cortex installation via its REST API. """
-    def __init__(self, job_queue, url="http://localhost:9001", api_token="",
-                 poll_interval=5, submit_original_filename=True,
+    def __init__(self, job_queue, url="http://localhost:9001", tlp="2",
+                 api_token="", poll_interval=5, submit_original_filename=True,
                  max_job_age=900, retries=5, backoff=0.5):
         """ Initialize the object.
 
         @param job_queue: The job queue to use from now on
         @type job_queue: JobQueue object
         @param url: Where to reach the Cortex REST API
         @type url: string
+        @param tlp: colour according to traffic light protocol
+        @type tlp: int ("red":3,"amber":2,"green":1,"white":0)
```

Comment:
Would you prefer to consistently use the `enum` child class?

Reply:
It's not a matter of preference: In a quick test, initializing a `Cortex` object with default parameter values throws an exception for me because of the default being a `str` which has no `.value` property. The same will happen if a user really supplies an `int` as documented. Alternatively we could eliminate the `.value` accessor and go with duck-typing or do explicit type-checking using `isinstance` and convert accordinly. Seems a lot of effort to me though.

Reply:
Seems like this is also fixed in the latest commit.